docker/Dockerfile

```diff
@@ -0,0 +1,89 @@
+FROM debian:stretch as conpot-builder
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Add non-free packages, needed for snmp-mibs-downloader
+RUN sed -i -e 's/main/main non-free contrib/g' /etc/apt/sources.list
+
+# Install dependencies
+RUN apt-get update -y -qq && apt-get install --no-install-recommends -y -qq \
+        ipmitool \
+        supervisor \
+        tcpdump \
+        git \
+        python3-pip \
+        python3 \
+        python3-dev \
+        python3-setuptools \
+        build-essential \
+        python3-scapy \
+        python3-cffi \
+        wget &&\
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Copy the app from the host folder (probably a cloned repo) to the container
+RUN useradd -m conpot
+WORKDIR /opt/
+RUN git clone https://github.com/mushorg/conpot.git
+RUN chown conpot:conpot -R /opt/conpot
+
+# Install Python requirements
+USER conpot
+ENV PATH=$PATH:/home/conpot/.local/bin
+WORKDIR /opt/conpot
+RUN pip3 install --user --no-cache-dir coverage
+RUN pip3 install --user --no-cache-dir -r requirements.txt
+
+# Run test cases
+RUN py.test -v
+
+# Install the Conpot application
+RUN python3.5 setup.py install --user --prefix=
+
+WORKDIR /home/conpot
+RUN tar -zcf conpot.tgz .local
+
+FROM debian:stretch
```

Comment:
can you try the alpine image? See [here](https://github.com/dtag-dev-sec/tpotce/blob/master/docker/conpot/Dockerfile#L1)

Reply:
I know alipne would help with size of image, I could try but this will need more time

Reply:
alpine would only help somehow as if we want to reduce size of image more changes need to be made, as things pulled by pip during compile are 200MB. I think I will do a clean up of previous things, we can then merge this one as it is usable and I will start working on another branch with reducing size of image. What do you think?

Reply:
Yeah, let's try lowering the overall size using one of the Alpine images (e.g. `python:3-alpine`)

Reply:
moved to alpine image